File: 진우/10weeks/1427. 소트인사이드/소트인사이드.cc
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    string N;
    cin >> N;  
    // 숫자를 문자열 형태로 입력받음
    // 예: 입력 "314159" → N = "314159"

    vector<int> arr;

    // 문자열의 각 문자를 숫자로 변환하여 저장
    for (char c : N) {
        arr.push_back(c - '0');  
        // '0'을 빼서 문자 → 정수 변환
        // 예: '3' - '0' = 3
    }

    // 내림차순 정렬 (큰 숫자부터)
    sort(arr.begin(), arr.end(), greater<>());
    // greater<>() : 비교 함수, 큰 값이 앞에 오도록 정렬

    // 정렬된 숫자 출력
    for (int i = 0; i < (int)arr.size(); i++) {
        cout << arr[i];
    }

    cout << "\n";  // 줄바꿈 출력

    return 0;
}

File: 진우/10weeks/1427. 소트인사이드/소트인사이드.py
N = input()  # 숫자를 문자열 형태로 입력받음 (예: "2143")

arr = []  # 각 자릿수를 저장할 리스트

# 입력된 문자열의 각 문자를 순회하며 정수로 변환하여 리스트에 추가
for i in range(len(N)):
    arr.append(int(N[i]))

# 숫자들을 내림차순으로 정렬
arr.sort(reverse=True)

# 정렬된 숫자들을 문자열로 변환하여 공백 없이 이어 붙여 출력
print("".join(map(str, arr)))
